services/gundeck/test/integration/API.hs

```diff
@@ -1047,10 +1085,13 @@ retryWhileN n f m =
     (const (pure . f))
     (const m)
 
-waitForMessage :: TChan ByteString -> IO (Maybe ByteString)
+waitForMessageRaw :: TChan WS.Message -> IO (Maybe WS.Message)
+waitForMessageRaw = System.Timeout.timeout 1000000 . liftIO . atomically . readTChan
```

Comment:
`1000000` means *1 Second*, right? Wouldn't this test become more stable if we would allow to wait for e.g. *3 Seconds*?

Reply:
sure. I've not seen these particular tests flake, but they could of course, and increasing timeout a little should be fine.